cmake/Zephyr-sdkConfigVersion.cmake

```diff
@@ -14,10 +14,15 @@ else()
   # Currently, this Zephyr SDK is expected to work with any Zephyr project
   # requiring this version or any older version.
   #
-  # In case this version is no longer backward compatible then this is the place
-  # to test, for example as
-  # set(ZEPHYR_MINIMUM_COMPATIBLE_VERSION 0.11)
-  # if(PACKAGE_FIND_VERSION VERSION_LESS ZEPHYR_MINIMUM_COMPATIBLE_VERSION)
+  # In case this version of Zephyr SDK is no longer backward compatible previous
+  # versions of Zephyr SDK this is the place to test if caller requested an
+  # older (in-compatible) revision.
```

Comment:
There are a few words missing here making it a little unclear what is being communicated.  Below is my best guess towards understanding the intent:

```
In case this verison of Zephyr SDK is no longer backwards compatible with
previous versions of Zephyr SDK, this is the place to test if the caller requested
an older (in-compatible) revision.
```

Reply:
incompatible isn't usually hypenated

Reply:
fixed, thanks for the suggestions.